di_website/templates/publications/partials/downloads.html

```diff
@@ -11,7 +11,13 @@
             <div class="orderReport__img" style="background-image: url({{cover_image.url}});"></div>
           {% endif %}
           <div class="orderReport__content">
-            <h2 class="type-m type-s--trailer">{{ page.download_report_title }}</h2>
+            <h2 class="type-m type-s--trailer">
+              {% if publication_downloads_title %}
```

Comment:
This won't ever be true ... I think you meant `page.publication_downloads_title`. Which makes me wonder whether you tested this. And if u tested it and found it fine, then this change was unnecessary.
Why did you changes this?? BTW, I'm using the comment you tagged in this PR as a reference for your changes.

If this is markup for the section in the screenshot below, then I think it was fine as it was.